src/imputors/standardize.jl

```diff
@@ -0,0 +1,84 @@
+"""
+    Standardize(; values)
+
+Standardize (or replace) various missing data values with `missing`.
+This is useful for downstream imputation methods that assume missing data is represented by
+a `missing`.
+
+Warning: In-place methods are only applicable for datasets which already `allowmissing`.
+
+# Keyword Arguments
+* `value::Tuple`: A tuple of values that should be considered `missing`
+
+# Example
+```jldoctest
+julia> using Impute: Standardize, impute
+
+julia> M = [1.0 2.0 -9999.0 NaN 5.0; 1.1 2.2 3.3 0.0 5.5]
+2×5 Array{Float64,2}:
+ 1.0  2.0  -9999.0  NaN    5.0
+ 1.1  2.2      3.3    0.0  5.5
+
+julia> impute(M, Standardize(; values=(NaN, -9999.0, 0.0)))
+2×5 Array{Union{Missing, Float64},2}:
+ 1.0  2.0   missing  missing  5.0
+ 1.1  2.2  3.3       missing  5.5
+```
+"""
+struct Standardize <: Imputor
+    values::Tuple
```

Comment:
I wonder if we should just have a function,
so that people can do things like coverting all numbers that are >10000 to missing, if that would indicate a sensor fault.
and then we have the tuple constructor creates a `v -> v in tuple`

Either way, we probably want to specialize on this as a type parameter.
Since if it is a tuple , it will let the compile specialize out checks to `in` that are of the wrong type.
and if its  a function then type stability

Reply:
I'll specialize on the tuple for now and I've made an issue for the function idea. https://github.com/invenia/Impute.jl/issues/78